demos/python_demos/action_recognition/action_recognition.py

```diff
@@ -41,6 +41,9 @@ def build_argparser():
                       help='Optional. Enable reading the input in a loop.')
     args.add_argument('-o', '--output', required=False,
                       help='Optional. Name of output to save.')
+    args.add_argument('-limit', '--output_limit', required=False, default=1000, type=int,
+                      help='Optional. Number of frames to store in output. '
+                           'If -1 is set, all frames will be stored.')
```

Comment:
Can you change from first conditional to zero? :) That's not just possible that if -1 is set then all frames may be are to be stored. That's a rule you describe.

Reply:
Done.